static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -120,7 +120,7 @@ export const init = (): Promise<void> => {
                 let npaFlag: boolean;
                 if (typeof state.tcfv2 !== 'undefined') {
                     // TCFv2 mode,
-                    npaFlag = Object.values(state.tcfv2).every(Boolean);
+                    npaFlag = Object.values(state.tcfv2.tcfData).includes(false);
                 } else {
                     // TCFv1 mode
                     npaFlag = Object.values(state).includes(false);
```

Comment:
I don't know about V1, but it would be interesting to know what happens in V1 if all purposes are denied consent.

Reply:
Just checked in V1, and it's all groovy. Deniying consent to all purpose gives an object with false values.